Megoldasok/VargaAdam/divisors1/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿Console.WriteLine("This program will write out numbers between a closed interval that can be divided by the divisor.");
+
+Console.WriteLine("What's the lower limit?");
+int lowerLimit = int.Parse(Console.ReadLine());
+Console.WriteLine("What's the upper limit?");
+int upperLimit = int.Parse(Console.ReadLine());
+Console.WriteLine("Add a divisor number:");
+int divisorNumber = int.Parse(Console.ReadLine());
+
+Console.WriteLine("The following numbers are in the (closed) interval you've given:");
+
+for (int i = lowerLimit; i <= upperLimit; i++)
+{
+    Console.Write(i);
+
```

Comment:
Es mi van akkor, ha a `lowerLimit>upperLimit`? akkor is kell mukodnie

Reply:
jó estét, kijavítottam.